### PR TITLE
Visual bug fixes for adoption of UHF for website - new

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

- Removed our header and footer components
- Adopted select UHF media query breakpoints so that our app zooms in and out smoothly, in line with the new header and footer
- Mobile side navigation is no longer in our app, that will be moving to UHF
- Updated index-devx.html to reflect recent updates to index.html, including shared style adjustment for UHF

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/fabric-website/src/pages/HomePage/HomePage.module.scss [hunk 1]
```diff
@@ -29,9 +29,9 @@
     background-position: center bottom;
     background-repeat: no-repeat;
     background-size: contain;
-    bottom: -10px;
+    bottom: -3px;
```
Comment (battletoilet) [inline apps/fabric-website/src/pages/HomePage/HomePage.module.scss]: These numbers feel a bit random, i would either create variables for what this and number below are or i would see if there is a way to reshape the containing elements so we don't have to pull this down -3px.

File: apps/fabric-website/src/styles/_mixins.scss [hunk 2]
```diff
@@ -38,9 +40,9 @@
   position: relative;
 
   &::after {
-    bottom: 0;
+    bottom: -1px;
```
Comment (battletoilet) [inline apps/fabric-website/src/styles/_mixins.scss]: Same here, i'm not sure why we need to have a negative value. I would try and make a variable for this or reshape the containing elements so everything fits perfectly.

File: apps/fabric-website/src/pages/HomePage/HomePage.module.scss (hunk 1 above)
Comment (lynamemi) [reply apps/fabric-website/src/pages/HomePage/HomePage.module.scss]: @battletoilet Yes, I was pixel pushing to get the hero and angle svgs to span the app, given I had to increase the width of the app to match UHF. I am all for reshaping in another way - let's sync up on that today.

Comment (battletoilet) [reply apps/fabric-website/src/pages/HomePage/HomePage.module.scss]: For now let's just store this value in a variable with a name that explains it a bit. We can always add this to the backlog and get to it later.

**Pull request merged**